certified-connectors/Impexium/apiDefinition.swagger.json

```diff
@@ -1130,34 +559,36 @@
         ]
       }
     },
-    "/api/v1/Individuals/{id}/Relationships": {
+    "/api/v1/Individuals/{ID}/CustomFields": {
       "post": {
-        "tags": ["Individuals"],
+        "tags": [
+          "Individuals"
+        ],
         "responses": {
           "200": {
-            "description": "200",
+            "description": "Success",
             "schema": {}
           }
         },
-        "summary": "Add Relationship to Individual",
-        "description": "Add Relationship to Individual.",
-        "operationId": "Add-relationship-to-individual",
+        "summary": "Update Custom Field Value",
```

Comment:
Please change to `Update Individual's Custom Field Value` as there is a different endpoint to update values for Organizations